src/pages/components/link-list.mdx

```diff
@@ -85,9 +86,18 @@ This variation displays links as a vertical list with a semibold heading, divide
 
 ![Image of Table of contents component](../../images/component/link-list/link-list-end-of-section-use-case.jpg)
 
+## Content guidance
+
+| Field | Field type | Required   | Cardinality | Max character size (English / translated) | Notes | 
+| -------- | --------- | ---------- | ---------- | ----------| ----------|
+| Headline | Short copy | Yes | 1 | XXXS (30 / 45)| - |
+| [CTA (text)](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/cta) | Component | Yes | 2-3 | - | - |
```

Comment:
If we are including functional props into the content guidelines then I think we need horizontal, and number of columns at the least.